config/routes.rb

```diff
@@ -2,6 +2,11 @@
 
   root controller: :index, action: :index
 
+  resources :shopping_carts, only: [:show], as: :cart do
+    resources :shopping_cart_items, only: [:create], as: :items
+  end
+  resources :shopping_cart_items, only: [:create], as: :items
```

Comment:
Why are you defining both a nested route and a stand alone route for creating `items`?